### PR TITLE
Replace content of file with lorem ipsum

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This looks good to me

**Review by ghost — COMMENTED** [review]
Can you change this paragraph and write it with a different style?

File: sample.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-This is our first test file in this repository
\ No newline at end of file
+
+What is Lorem Ipsum?
```
Comment (getintotechlink) [inline sample.txt]: Can you replace this with "Lorem Ipsum?"

File: sample.txt [hunk 2]
```diff
@@ -1 +1,4 @@
...
+
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
```
Comment (getintotechlink) [inline sample.txt]: Can you change this paragraph and write it with a different style?

File: sample.txt (hunk 1 above)
Comment (getintotechlink) [reply sample.txt]: No problem. i have done this.

File: sample.txt (hunk 2 above)
Comment (getintotechlink) [reply sample.txt]: I have done this

**Pull request merged**